common/src/main/java/io/netty/util/concurrent/EventExecutorGroup.java

```diff
@@ -116,4 +110,12 @@
 
     @Override
     ScheduledFuture<?> scheduleWithFixedDelay(Runnable command, long initialDelay, long delay, TimeUnit unit);
+
+    /**
+     * {@link EventExecutorGroup} can be wrapped to extend and enhance behavior. This method will unroll the wrapping
+     * and return the underlying {@link EventExecutorGroup}.
+     *
+     * @return {@code this} if the object is not wrapped or underlying {@link EventExecutorGroup} instance if wrapped.
+     */
+    EventExecutorGroup unRollWrapping();
```

Comment:
The idea came from this:

- https://github.com/line/armeria/blob/master/src/main/java/com/linecorp/armeria/server/Service.java#L101
- https://github.com/line/armeria/blob/master/src/main/java/com/linecorp/armeria/server/DecoratingService.java#L110